enhancements/cluster-logging/cluster-logging-move-init-to-es-operator.md

```diff
@@ -70,6 +70,7 @@ The following changes will define how the elasticsearch-operator executes init p
 * Adding additional `ClusterConditionType` for each action 
 * Executing given scripts with the image that contains all the init logic against the ES API node
 * Initialization logic will be executed from outside the ES container by creating a `Job`
+* ES cluster will be become available **after** the initialization. During the initalization cluster won't be visible for extrnal service.
```

Comment:
@lukas-vlcek does this address your concern?

Reply:
@igor-karpukhin @ewolinetz Sounds goof on high level :-) Definitely good start.

Reply:
I mean, I can imagine it might be useful to have some mechanism how to check if initialization happened correctly but this can wait (let's not make it more complicated now, may be that would be over engineering anyway...).